pkg/generate/app/pipeline.go

```diff
@@ -200,6 +203,15 @@ func (p *Pipeline) Objects(accept, objectAccept Acceptor) (Objects, error) {
 		}
 		if objectAccept.Accept(repo) {
 			objects = append(objects, repo)
+		} else {
+			// if the image stream exists, if possible create the imagestream tag referenced if that does not exist
+			tag, err := p.Image.ImageStreamTag()
+			if err == nil {
+				// if err just move on with creating image stream
```

Comment:
with?  without?

Reply:
also should probably log any error.

Reply:
yeah the comment was a leftover from a prior implementation ...  removing

adding error log as well ... part of next push

Reply:
sorry ... actually, now I thinks it makes sense to return an error like we do elsewhere in this method ... doing that